### DISCUSSION TITLE
Devtool Services for webdriver IO, Not able to see methods for browser objects

### DISCUSSION THREAD

**Discussion by kavita32** [webdriverio/webdriverio#7920] [Community]
Hi , I have installed devtool services for webdriver IO listed here https://webdriver.io/docs/devtools-service/
But still I am not able to see methods like browser.enablePerformanceAudits(), browser.getPerformanceScore() for browser object, Has anyone encountered this problem too? Any solutions for this please.

**Comment by christian-bromann** [comment]
Note that this service only works for local testing on Firefox and Chrome.

**Reply by kavita32** [reply]
We are using this service for Local testing for Chrome Only.
Do you have any idea about this solution.

**Reply by kavita32** [reply]
i have installed this library locally and following the setup and example as given in the webdriverio page, methods aren't available in the browser object
